Libplanet/Tx/Transaction.cs

```diff
@@ -181,6 +156,31 @@ private Transaction(SerializationInfo info, StreamingContext context)
             }
         }
 
+        // ReSharper disable once UnusedMember.Local
```

Comment:
Is this diff necessary?

Reply:
No, I doesn't intended it. I'll amend it.